plugin/evm/vm.go

```diff
@@ -343,6 +343,12 @@ type VM struct {
 	// Avalanche Warp Messaging backend
 	// Used to serve BLS signatures of warp messages over RPC
 	warpBackend warp.Backend
+
+	p2pSender             p2p.AppSender
+	ethTxGossipHandler    p2p.Handler
+	atomicTxGossipHandler p2p.Handler
+	ethTxGossiper         gossip.Gossiper
+	atomicTxGossiper      gossip.Gossiper
```

Comment:
Keeping these interfaces here lets us override these with noop implementations during testing.

Reply:
could we add this as a comment?